glib/gobject-sys/src/lib.rs

```diff
@@ -2505,6 +2515,9 @@ extern "C" {
         strict_validation: gboolean,
     ) -> gboolean;
     pub fn g_param_value_defaults(pspec: *mut GParamSpec, value: *const GValue) -> gboolean;
+    #[cfg(any(feature = "v2_74", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_74")))]
+    pub fn g_param_value_is_valid(pspec: *mut GParamSpec, value: *const GValue) -> gboolean;
```

Comment:
Needs manual implementation on `ParamSpec`: `fn value_is_valid(&self, value: &Value) -> bool`

Reply:
Done, though not sure how correct it is